alacritty_terminal/src/term/mod.rs

```diff
@@ -1468,6 +1761,7 @@ impl<T: EventListener> Handler for Term<T> {
                 self.tabs.clear_all();
             },
         }
+        self.is_fully_damaged = true;
```

Comment:
Why does this require a redraw? Shouldn't the content be unaffected by `clear_tabs`?